### DISCUSSION TITLE
How to set different colors for bigwig tracks?

### DISCUSSION THREAD

**Discussion by typekey** [GMOD/jbrowse-components#1487] [Q&A] [answered]
Hello, I am going to set different colors for different bigwig tracks, similar to UCSC Genome Browser, but I don’t seem to find relevant options in the manual.

**Comment by cmdcolin** [comment]
This is a bit involved but this could be a way to do it via the command line


`jbrowse add-track yourfile.bw --config '{"displays": [{"displayId": "coverage_1", "type":"LinearWiggleDisplay", "renderers": {"XYPlotRenderer": {"color": "green", "type": "XYPlotRenderer"} } }]}' --load copy`

We currently have the "color" as part of the renderer instead of the track config, which makes it a bit more deeply nested than you might expect....

**Reply by typekey** [reply]
Thanks you!
your help is great

**Comment by garrettjstevens** [comment, marked as answer]
You can also change the color of the BigWig track using the graphical config editor. You can access this by clicking "Settings" in the menu of the track selector:

![image](https://user-images.githubusercontent.com/25592344/99996084-82e05f00-2d78-11eb-87f8-f4a6dd629301.png)

The "Settings" option will be available if you are configuring JBrowse with the JBrowse CLI's `admin-server` (see [here](https://jbrowse.org/jb2/docs/quickstart_gui)),

![image](https://user-images.githubusercontent.com/25592344/99996798-7c9eb280-2d79-11eb-8b5e-f5ed8968980e.png)

If you want to experiment with different settings before changing your config file (and without running `admin-server`), I recommend making a copy of the track (see [here](https://jbrowse.org/jb2/docs/user_guide#editing-track-configs)) and editing the copy. The copy of the track exists only in the current session of JBrowse and is not part of your config file.

**Reply by typekey** [reply]
Thanks, this way is  very useful!

**Comment by cmdcolin** [comment]
Example for changing feature color on a gff3tabix/svgfeaturerenderer track

    jbrowse add-track yourfile.sort.gff3.gz --config '{"displays": [{"displayId": "basicdisplay","type":"LinearBasicDisplay", "renderer": {"color1": "green", "type": "SvgFeatureRenderer"}}]}' --load copy
